src/clarity-icons/shapes/basic-shapes.ts

```diff
@@ -4,7 +4,8 @@
  * The full license information can be found in LICENSE in the root directory of this project.
  */
 
-export const BasicShapes: any = {
+/* tslint:disable:max-line-length */
+export const basicShapes: any = {
```

Comment:
I thought we weren't going to use "basic". Maybe I am remembering incorrectly?

But I thought we were going to use "essential".

@Shijir + @lil-kim